docs_build/tutorials_templates/task_workflows/item_status/mds.py

```diff
@@ -9,25 +9,36 @@ def func1():
     ### 1. Set status on an item in task
     """
 
+
 def func2():
     """
     ### 2. Set status on multiple items
+    #### 1. using dataset and filter (this way recommended of the items is not in the same task and items that include in only one task)
```

Comment:
Using dataset and filter (recommended for using with multiple items from different tasks)